### ISSUE TITLE
flavors.py still talks about getObjectAt

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4272]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#4272|
|Type|defect|
|Created|2010-02-11 20:10:25Z|

The docstring for `twisted.spread.flavors.Root` refers readers to `pb.getObjectAt`, a long-since deleted API.  We should drop this reference.

Attachments:

* [removed_reference.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/7df/7df0d5be644e6aa2dfed8c117fc5c4cd61d489c0/421673d2c023ff52076d76df50b58daff6066e93.patch) (425 bytes) - added by jtanis on 2010-02-22 01:22:27Z - Removed pb.getObjectAt reference.

<details><summary>Searchable metadata</summary>

```
trac-id__4272 4272
type__defect defect
reporter__exarkun exarkun
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__pb pb
keywords__documentation__easy documentation, easy
time__1265919025000000 1265919025000000
changetime__1266877539000000 1266877539000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|<img alt="jtanis's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|jtanis removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|<img alt="terrycojones's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_2"></a>|terrycojones commented|
|-|-|

A patch of great beauty.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun set status to `closed`|
|-|-|

(In [28422]) Apply removed_reference.patch

Author: jtanis
Reviewer: terrycojones
Fixes: [#4272](https://github.com/twisted/twisted/issues/4272)

Remove outdated reference to `pb.getObjectAt`.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
